front/src/components/login.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+
+export default class Login extends Component {
+    constructor(props){
+        super(props);
+        this.state={
```

Comment:
pls format it properly (`this.state = {`, so add space) and the same everywhere
In intellij: CTL + ALT + L (https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html)